lib/getProcessStat.js

```diff
@@ -26,9 +35,9 @@ function getProcessStat(pid, callback)
        'cguest_time', 'start_data', 'end_data', 'start_brk', 'arg_start',
        'arg_end', 'env_start', 'env_end', 'exit_code')
 
-   callback(null, result)
+   return callback(null, result)
```

Comment:
The return Statement ends the execution in this code block so it wont execute anything after that, you're right that just a convenience but I think this should be done